### PR TITLE
Adds Gnollhack and adds Notdnethack as first ascension

### PR DESCRIPTION
**Pull request by nethack-cleaner**
Adds Gnollhack as variant and updates Notdnethacks list of ascension types and makes them first ascension listed.

The file schema/addgnoll.sql
will add the records for Gnollhack tracking.

Your max(logfiles_i) may be different than mine, in which case you would need to adjust that field prior to running it.

### CODE REVIEW

File: nhdb-feeder.pl [hunk 1]
```diff
@@ -1453,6 +1453,11 @@ sub sql_player_name_map
     #--- parse log
 
       my $parsed_line = parse_log($log, $xlog_line);
+      if ($log->{'variant'} eq 'gnoll') {
```
Comment (aoeixsz4) [inline nhdb-feeder.pl:1456]: what the heck is this part about?

File: schema/addgnoll.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
+insert into logfiles (logfiles_i, descr, server, variant, logurl, localfile, dumpurl, oper, static, tz, fpos, lines)values (2025, 'gnollhack.org', 'hdf', 'gnoll', 'https://www.hardfought.org/xlogfiles/gnollhack/xlogfile', 'hdf.gnoll.log','https://www.hardfought.org/userdata/%X/%x/gnollhack/dumplog/%s.gnoll.txt', 't', 'f', 'UTC', 0, 0);
```
Comment (aoeixsz4) [inline schema/addgnoll.sql]: please add these to the already existing sources schema instead of adding a new flie

**Review by aoeixsz4 — CHANGES_REQUESTED** [review]

File: schema/addgnoll.sql (hunk 2 above)
Comment (aoeixsz4) [reply schema/addgnoll.sql]: and just increase sequence number from the last double or triple-digit value

**Comment by nethack-cleaner** [comment]
Replacing with revision.

**Pull request closed without merging**